Configuration/Services.yaml

```diff
@@ -12,9 +12,15 @@ services:
     public: true
   B13\Newspage\Filter\CategoriesFilter:
     public: true
+  B13\Newspage\Controller\LayoutModuleEditController:
+    public: true
```

Comment:
hm, why should this be a public service?

Reply:
Not needed anyways, as the controller is tagged with `Controller`

Reply:
Removed the public service